### ISSUE TITLE
Gii module generator fails to escape apostrophes when generating labels from comments

### ISSUE THREAD

**Issue by MarcoMoreno** [yiisoft/yii#3486]
Gii's module generator (Yii 1.1.15) fails to escape any apostrophes that may exist in a db column's comments when generating labels, resulting in a syntax error:

```
/**
 * @return array customized attribute labels (name=>label)
 */
public function attributeLabels()
{
    return array(
        'AuthorID' => 'Author',
        'FirstName' => 'Author's First Name',
        'LastName' => 'Author's Last Name',
    );
}
```

Instead, this should be:

```
/**
 * @return array customized attribute labels (name=>label)
 */
public function attributeLabels()
{
    return array(
        'AuthorID' => 'Author',
        'FirstName' => 'Author\'s First Name',
        'LastName' => 'Author\'s Last Name',
    );
}
```

**Comment by NovikovViktor** [comment]
Hello, @samdark ,it seems like this issue already solved:
 -  [commit](https://github.com/yiisoft/yii/commit/a0a9165a3e6f5000c2f5e5f0adb4f01fb6163bb0)
 -  [template](https://github.com/yiisoft/yii/blob/master/framework/gii/generators/model/templates/default/model.php#L101)

But maybe just need to substitute ```str_replace()``` to ```addslashes()```.

**Comment by samdark** [comment]
Won't be fixed in 1.1.
